scripts/integration/docker-compose.loki.yml

```diff
@@ -1,10 +1,8 @@
 services:
 
   loki:
-    image: grafana/loki:${LOKI_VERSION:-2.1.0}
-    command: -config.file=/etc/loki/loki-config.yaml
-    volumes:
-      - ${PWD}/tests/data:/etc/loki
+    image: grafana/loki:${LOKI_VERSION:-2.4.1}
+    command: -config.file=/etc/loki/local-config.yaml -auth.enabled=true
```

Comment:
I moved the `auth.enabled` to a flag here so we can just use the default `local-config.yaml` and delete our custom one which the newer version of Loki was not starting with.